### ISSUE TITLE
Add an option to ignore private property names start with `_`

### ISSUE THREAD

**Issue by Goooler** [pinterest/ktlint#2326]
## Expected Behavior

We can ignore private properties like `Foo._foo`, which start with `_`

```kotlin
class Foo {
    private var _foo = "foo"

    fun bar() {
        _foo = "bar"
    }
}
```

## Current Behavior

Using CLI:
```sh
ktlint Foo.kt   
/Users/goooler/StudioProjects/lawnicons/Foo.kt:2:17: Property name should start with a lowercase letter and use camel case (standard:property-naming)

Summary error count (descending) by rule:
  standard:property-naming: 1
```

EditorConfig file:
```editorconfig
root = true

[*]
charset = utf-8
indent_size = 4
indent_style = space
insert_final_newline = true
trim_trailing_whitespace = true

[*.{kt,kts}]
ij_kotlin_imports_layout = *
ij_kotlin_allow_trailing_comma = true
ij_kotlin_allow_trailing_comma_on_call_site = true
ktlint_code_style = intellij_idea
```

## Additional information
* Current version of ktlint: `1.0.1`

**Comment by Goooler** [comment]
Although we can add `ktlint_standard_property-naming = disabled` rule to suppress all property checks, I just want to ignore private properties here.

**Comment by Goooler** [comment]
I missed something refs to `Foo._foo`, that sounds make sense.

```kotlin
class Foo {
    private var _foo = "foo"

    val foo: String get() = _foo

    fun bar() {
        _foo = "bar"
    }
}
```
